lib/timeline/component/ItemSet.js

```diff
@@ -1109,6 +1120,26 @@ class ItemSet extends Component {
     }
   }
 
+  
+  /**
+   * Control an item by its id
+   * @param {string | number} id
+   */
+  controlItem(id) {
+    const item = this.itemsData.get(id);
+
+    if (item) {
+      // confirm deletion
+      this.options.onControl(item, item => {
+        if (item) {
+          // control by id here, it is possible that an item has no id defined
```

Comment:
please explain again, I don't understand the comment.
Why is this if statement even defined? It doesn't do anything.

Reply:
You´re right this is not needed. I have changed this part, now